app/src/molecules/Modal/OnDeviceDisplay/Modal.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import {
+  BORDERS,
+  COLORS,
+  Flex,
+  SPACING,
+  DIRECTION_COLUMN,
+  JUSTIFY_CENTER,
+} from '@opentrons/components'
+import { BackgroundOverlay } from '../../BackgroundOverlay'
+import { ModalHeader } from './ModalHeader'
+
+import type { ModalHeaderProps, ModalSize } from './types'
+
+interface ModalProps {
+  //  clicking anywhere outside of the modal closes it
```

Comment:
Let's use /** */ for props' description.
Not 100% sure but Storybook Doc follows JSDoc's style which means Storybook Doc cannot recognize `//` as a prop description.  

```suggestion
  /** clicking anywhere outside of the modal closes it */
```